lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -124,41 +124,65 @@ sub create_db {
         '
     );
```

Comment:
Shouldn't there be a die() unless the table could be created?

Reply:
I'd rather avoid dying if I can. I don't see what could prevent the table from being created at this point in the code. Especially since we explicitly check if it does not exist.

Reply:
I do not understand the problem of die(). It would catch bugs that lets the code continue even if the table cannot be created.

Reply:
I added a die after all table creation in case something happens.